docs/sources/auth/overview.md

```diff
@@ -14,21 +14,20 @@ weight = 1
 Grafana provides many ways to authenticate users. Some authentication integrations also enable syncing user
 permissions and org memberships.
 
-## OAuth Integrations
+Here is a table showing all supported authentication providers and the features available for them. [Team sync]({{< relref "../enterprise/team-sync.md" >}}) and active sync are only available in Grafana Enterprise.
+
+Provider | Support | Role mapping | Team sync<br>*(Enterprise only)* | Active sync<br>*(Enterprise only)*
+-------- | :-----: | :----------: | :-------: | :---------: 
+[Auth Proxy]({{< relref "auth-proxy.md" >}})       | v2.1+ | - | v6.3+ | - 
+[Azure AD OAuth]({{< relref "azuread.md" >}})      | v6.7+ | (v6.7+) | v6.7+ | - 
+[Generic OAuth]({{< relref "generic-oauth.md" >}}) | v4.0+ | v6.5+ | - | - 
+[GitHub OAuth]({{< relref "github.md" >}})         | v2.0+ | - | v6.3+ | -
+[GitLab OAuth]({{< relref "gitlab.md" >}})         | v5.3+ | - | v6.4+ | -
+[Google OAuth]({{< relref "google.md" >}})         | v2.0+ | - | - | - 
+[LDAP]({{< relref "ldap.md" >}})                   | v2.1+ | v2.1+ | (v6.3+) | v6.3+
+[Okta OAuth]({{< relref "okta.md" >}})             | v7.0+ | v7.0+ | v7.0+ | - 
+[SAML]({{< relref "../enterprise/saml.md" >}}) (Enterprise only)    | v6.3+ | v7.0+ | v7.0+ | - 
```

Comment:
@torkelo @alexanderzobnin It would be great if you have some time to review this. I'm unsure of the two versions between parenthesis and also I might have missed the role mapping feature for some providers.

Reply:
For the `*`, I was testing on the old layout so I didn't notice but it's working now (just needed a space before).

I replaced the version. I searched a bit more on LDAP Team Sync, I didn't find any clear information but the file seems to have been created around 5.3 so it makes sense!